website/src/components/features.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import BasicInstall from "../components/basicInstall"
+
+const Feature = (props) =>
+  <div className="row padding--md feature-row">
+    <div className="col col--1"/>
+    <div className="col col--6" key="img">
+      {!props.image ? "" : <img src={props.image}/>}
+    </div>
+    <div className="col col--5" key="text">
+      <div className="padding--md"/>
+      <h3>{props.title}</h3>
+      <div className="padding--sm"/>
+      {props.children}
+    </div>
+    <div className="col col--1"/>
+  </div>
+
+const FeatureList = [
+    <Feature image="img/envs.gif" title="Scala versions, dependencies and JVMs" 
+      key="versions" education="true" simple_projects="true" scripting="true" prototyping="true"> 
+      <p>
+        scala-cli is built on top of coursier
+        <br/>
+        This allow us to manage Scala versions, dependencies and JVMs so you can test your code in different environments by changing single option.
+      </p>
+
+      <p>
+        scala-cli install all its depenendencies
+        <br/>
+        No need to fluff with installing JVM or setting up PATH. 
+      </p>
+    </Feature>,
+    <Feature image="img/envs.gif" title="Universal tool" key="universal" 
+      simple_projects="true" scripting="true" prototyping="true">
+      <p>
+        If you want to use older <b>version of Scala</b> or
+        run your code in <b>JS</b> or <b>Native</b> environments we've got you covered.
+        <br/>
+        <i>some additional <a href="TODO?">setup</a> may be required for JS and Native</i>
```

Comment:
For scalaJs, the Node is required - https://github.com/VirtuslabRnD/scala-cli-features/issues/6

Reply:
Could you add documentation what and how should be installed to js section in the documentation and link it here? Do we need any specific setup for native?